syntax = "proto3";

service FileMonitorService {
  rpc SendFileMetadata (FileMetadata) returns (Ack);
}

message FileMetadata {
  string file_name = 1;
  string action = 2;
  string timestamp = 3;
  bool flag = 4;
}

message Ack {
  bool success = 1;
}
